packages/template-retail-react-app/app/pages/social-login-redirect/index.jsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {Fragment} from 'react'
+import {FormattedMessage} from 'react-intl'
+import {Box, Container, Stack, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {BrandLogo} from '@salesforce/retail-react-app/app/components/icons'
+
+const SocialLoginRedirect = () => {
+    return (
+        <Box data-testid="login-redirect-page" bg="gray.50" py={[8, 16]}>
+            <Container
+                paddingTop={16}
+                width={['100%', '407px']}
+                bg="white"
+                paddingBottom={14}
+                marginTop={8}
+                marginBottom={8}
+                borderRadius="base"
+            >
+                <Fragment>
+                    <Stack justify="center" align="center" spacing={8} marginBottom={8}>
+                        <BrandLogo width="60px" height="auto" />
+                        <Text align="center" fontSize="xl" fontWeight="semibold">
+                            <FormattedMessage defaultMessage="Authenticating..." />
+                        </Text>
+                        <Text align="center" fontSize="m">
+                            <FormattedMessage
+                                defaultMessage="If you are not automatically redirected, click <link>this link</link> to proceed."
+                                values={{
+                                    link: (chunks) => (
+                                        <a
+                                            href="/account"
```

Comment:
I'm assuming `this link` should navigate to the secure account page. If login is successful, the user will see their user information in the account page.

Reply:
I'd personally expect this to have a return_to functionality, as in, I navigate to a PDP, I click the profile icon at the top of the window, the modal opens, I click Google, it brings me to Google, I sign in, I'm sent back to see this authenticating modal and then dropped back to the original PDP that I came from.

Reply:
Hey @bredmond-sf I did some testing and found that the current behavior of the template retail react app doesn't follow this flow. If I navigate to a PDP, then open the login modal and sign in, I am redirected to the account page. If I am on a PDP and I log out, I am redirected to the login page.